Package.swift

```diff
@@ -238,22 +199,18 @@ var dependencies: [Package.Dependency] = [
     // ~~~ Swift Collections  ~~~
     .package(url: "https://github.com/apple/swift-collections.git", from: "1.0.1"),
 
-    // ~~~ SSWG APIs ~~~
+    // ~~~ Observability ~~~
     .package(url: "https://github.com/apple/swift-log.git", from: "1.0.0"),
     // swift-metrics 1.x and 2.x are almost API compatible, so most clients should use
     .package(url: "https://github.com/apple/swift-metrics.git", "1.0.0" ..< "3.0.0"),
     .package(url: "https://github.com/apple/swift-service-discovery.git", from: "1.0.0"),
-]
-
-// ~~~ only for DistributedActorsGenerator ~~~
-dependencies += [
-    .package(url: "https://github.com/apple/swift-argument-parser", .upToNextMinor(from: "0.3.2")), // not API stable, Apache v2
+    .package(url: "https://github.com/apple/swift-distributed-tracing.git", from: "0.3.0"),
 ]
 
 // swift-syntax is Swift version dependent, and added as such below
 #if swift(>=5.6)
 dependencies.append(
-    // Works with: swift-PR-39654-1170.xctoolchain
+      // Works with: swift-PR-39654-1170.xctoolchain
```

Comment:
is swift syntax still needed? since no code generator

Reply:
Right, no need for it -- thanks for spotting :)